apple/LayoutReanimation/REAScreensHelper.h

```diff
@@ -17,5 +18,10 @@
 + (REAUIView *)getScreenWrapper:(REAUIView *)view;
 + (int)getScreenType:(REAUIView *)screen;
 + (bool)isRNSScreenType:(REAUIView *)screen;
++ (bool)isStackChanged:(REAUIView *)view;
```

Comment:
What does this method do? What does it mean that stack ~~was~~ has been changed?